app/src/main/java/com/beemdevelopment/aegis/ui/MainActivity.java

```diff
@@ -105,6 +106,7 @@ protected void onCreate(Bundle savedInstanceState) {
         _loaded = false;
 
         if (savedInstanceState != null) {
+            _recreate = savedInstanceState.getBoolean("isReCreate");
```

Comment:
Do we really need to store this in the instance state bundle, or can we just directly assign ``true`` here? That seems equivalent and shorter.

Reply:
Enough i didn't think that much.